SofaKernel/modules/SofaLoader/CMakeLists.txt

```diff
@@ -10,15 +10,15 @@ set(HEADER_FILES
     ${SOFALOADER_SRC}/initSofaLoader.h
     ${SOFALOADER_SRC}/BaseVTKReader.h
     ${SOFALOADER_SRC}/BaseVTKReader.inl
-    ${SOFALOADER_SRC}/MeshObjLoader.h
+    ${SOFALOADER_SRC}/MeshOBJLoader.h
```

Comment:
TODO: keep MeshObjLoader.h as a forward header + use DEPRECATED_HEADER macro in it

Reply:
Deprecation period: v21.12 -> v22.12